test/language/class/definition/ClassMethod_restricted-properties.js

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2015 Caitlin Potter. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*--- 
+ description: >
+    Functions created using MethodDefinition syntactic form do not
+    have own properties "caller" or "arguments", but inherit them from
+    %FunctionPrototype%.
+---*/
+
+class Class {
+  method() {}
```

Comment:
I guess `get PropertyName ( ) { FunctionBody }` and the setter variant are MethodDefinitions, so they're covered... adding tests